proto/p4/config/p4info.proto

```diff
@@ -80,26 +78,29 @@ message Table {
   repeated int32 action_ids = 3;
   // 0 (default value) means that the table does not have a const default action
   int32 const_default_action_id = 4;
+  // a table may have a const default action, whose action parameter values can
+  // be changed at runtime. However, in most cases a table's action entry is set
+  // in the P4 program and cannot be changed, which is what this boolean flag
+  // indicates.
+  bool has_const_default_action_entry = 5;
   // P4 id of the "implementation" for this table (e.g. action profile id); 0
   // (default value) means that the table is a regular (direct) match table. As
   // of today, only action profiles are supported but other table
   // implementations may be added in the future
-  int32 implementation_id = 5;
+  int32 implementation_id = 6;
   // ids of the direct resources (if any) attached to this table; for now this
   // includes only direct counters and direct meters, but other resources may be
   // added in the future
-  repeated int32 direct_resource_ids = 6;
-  int64 size = 7;  // number of entries in table
+  repeated int32 direct_resource_ids = 7;
+  int64 size = 8;  // number of entries in table
+  bool with_entry_timeout = 9;  // entry ageing is enabled for table
```

Comment:
"Entry Aging" seem new (at least to me). Is this coming from P4_16? Seems like we need to discuss implications on P4Runtime (I would imagine controller specifying the age).